G3Client/controllers/controllers_gui/UserPortalOfBranchManagerController.java

```diff
@@ -93,7 +93,7 @@ public void getEditCustomerInfo(ActionEvent event) {
 	@FXML
 	public void getSupplierReg(ActionEvent event) {
 		((Node) event.getSource()).getScene().getWindow().hide(); // hiding primary window
-		SupplierRegistrationScreenController SRCC = new SupplierRegistrationScreenController();
+		SupplierWorkerRegistrationScreenController SRCC = new SupplierWorkerRegistrationScreenController();
```

Comment:
change SupplierWorkerRegistrationScreenController SRCC = new SupplierWorkerRegistrationScreenController();
to SupplierWorkerRegistrationScreenController supplierWorkerRegistrationScreenController = new SupplierWorkerRegistrationScreenController();